tests/test_console_xss_contract.py

```
@@ -0,0 +1,147 @@
+                trend_bars = page.locator("#trend .bar")
+                assert trend_bars.count() == 1
+
+                aria_label = trend_bars.first.get_attribute("aria-label")
+                title = trend_bars.first.get_attribute("title")
+                assert aria_label == "<img onerror=alert('date_xss')>: <img onerror=alert('block_xss')> blocking"
+                assert title == "<img onerror=alert('date_xss')>: <img onerror=alert('block_xss')> blocking"
+            finally:
+                try:
+                    browser.close()
+                except Exception:
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)